jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -1031,6 +1022,19 @@ def run(self):
                             # Copy files from filesystem
                             dark_files, not_copied = copy_files(new_filenames, self.data_dir)
 
+                            # Check that there were no problems with the file copying. If any of the copied
```

Comment:
@bhilbert4 this looks good to me, do we have an idea of why the files we are copying over are being truncated? Is it something to deal with MAST or could it be some sort of issue with our connection to the storage?

Reply:
Not sure. The files I noticed this for are all large >=4Gb, so maybe it's a connection issue? But I had never seen it before.

Reply:
Cool, to get this back up and running I think this is the best solution we have right now but I we should do some more thinking on this in the future.